fusedrug/data/tokenizer/modulartokenizer/create_multi_tokenizer.py

```diff
@@ -29,6 +29,7 @@ def test_tokenizer(
     mode: Optional[str] = "",
     input_strings: Optional[List] = None,
     on_unknown: Optional[str] = "warn",
+    verbose: bool = False,
```

Comment:
Maybe let's keep to the modular tokenizer verbosity format - integer (0, 1,...), so verbose=0 will be the same as your verbose - False, and verbose of 1 and above - your True?

Reply:
This will also make things simpler down the road when we call encode_list